### PR TITLE
Updated microsoft-azure-attestation's readme.md for the integration with Occlum

### PR DESCRIPTION
**Pull request by olkroshk**
## Purpose
Updated README.md for the integration sample code for doing Microsoft Azure Attestation in Occlum

## Does this introduce a breaking change?

```
[x] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[x] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
N/A

## What to Check
Verify that the updates provides clear and relevant information.

### CODE REVIEW

**Comment by qzheng527** [comment]
LGTM

File: README.md [hunk 1]
```diff
@@ -40,17 +40,5 @@ See a code sample to manage trusted signing certificates and keys for Isolated p
 
 - [EGo's code sample using Azure Attestation](https://github.com/edgelesssys/ego/tree/master/samples/azure_attestation)
 - [Azure Attestation integration in SCONE platform](https://github.com/scontain/scone-azure-integration)
+- [Sample code for doing Microsoft Azure Attestation in Occlum](https://github.com/occlum/occlum/tree/master/demos/remote_attestation/azure_attestation)
```
Comment (gkostal) [inline README.md]: _"doing"_ - this is a vague word.  What specifically is performed?  Would a more precise description be "Sample code to generate and verify an SGX quote using Microsoft Azure Attestation in Occlum"?  Or perhaps "Azure Attestation integration in Occlum"?

Comment (olkroshk) [reply README.md]: updated

**Comment by mssindhurid** [comment]
@kroshkina-ms  - "Azure Attestation integration in Occlum" looks good to me too

**Pull request merged**